src/main/webapp/resources/js/pages/AdminUsersPage.jsx

```diff
@@ -0,0 +1,158 @@
+import React, { useContext } from "react";
+import { render } from "react-dom";
+import { PageWrapper } from "../components/page/PageWrapper";
+import {
+  PagedTable,
+  PagedTableContext,
+  PagedTableProvider
+} from "../components/ant.design/PagedTable";
+import { setBaseUrl } from "../utilities/url-utilities";
+import { AddNewButton } from "../components/Buttons/AddNewButton";
+import { dateColumnFormat } from "../components/ant.design/table-renderers";
+import { Button, Checkbox } from "antd";
+import { EditOutlined } from "@ant-design/icons";
+import { setUsersDisabledStatus } from "../apis/users/users";
+
+function UsersTable() {
+  const { updateTable } = useContext(PagedTableContext);
+
+  function updateUser(user) {
```

Comment:
This should also display a notification when it's been updated.

Reply:
The table provides instant feedback by updating the status of the checkbox.  Nothing more is needed.

Reply:
Shouldn't the user have to confirm the change in that case? I'm just thinking about if a user accidentally disables someones account and doesn't realize it as no feedback was provided

Reply:
Because it is a simple task and easily undo-able, it does not need a confirmation.  Only complex tasks or ones that cannot be undone need confirmation.